app/src/main/res/layout/activity_booking_detail.xml

```diff
@@ -394,6 +299,177 @@
+                                    android:text="Left"
+                                    android:textColor="@color/color_white"/>
+                            </RelativeLayout>
+                        </android.support.v7.widget.CardView>
+
+                        <android.support.v7.widget.CardView
+                            android:layout_width="0dp"
+                            android:layout_height="wrap_content"
+                            android:layout_marginRight="@dimen/dp_3"
+                            android:layout_weight="1"
+                            car_view:cardElevation="@dimen/dp1">
+
+                            <RelativeLayout
+                                android:layout_width="match_parent"
+                                android:layout_height="@dimen/dp_80"
+                                android:onClick="@{()->viewModel.pickImage(SideCapture.SIDE_RIGHT)}">
+
+                                <ImageView
+                                    android:layout_width="match_parent"
+                                    android:layout_height="match_parent"
+                                    android:scaleType="centerCrop"
+                                    bind:imageUrl="@{viewModel.pathImageSideRight}"
+                                    tool:ignore="ContentDescription"/>
+
+                                <ImageView
+                                    android:layout_width="wrap_content"
+                                    android:layout_height="wrap_content"
+                                    android:layout_centerInParent="true"
+                                    android:alpha="@dimen/float_0.6"
+                                    android:visibility="@{viewModel.pathImageSideRight == null ? View.VISIBLE : View.GONE}"
+                                    bind:srcCompat="@drawable/ic_camera_24dp"
+                                    tool:ignore="ContentDescription"/>
+
+                                <TextView
+                                    android:layout_width="match_parent"
+                                    android:layout_height="wrap_content"
+                                    android:layout_alignParentBottom="true"
+                                    android:alpha="@dimen/float_0.6"
+                                    android:background="@color/color_transparent"
+                                    android:gravity="center"
+                                    android:padding="@dimen/dp_1"
+                                    android:text="Right"
+                                    android:textColor="@color/color_white"/>
+                            </RelativeLayout>
+                        </android.support.v7.widget.CardView>
+
+                        <android.support.v7.widget.CardView
+                            android:layout_width="0dp"
+                            android:layout_height="wrap_content"
+                            android:layout_marginRight="@dimen/dp_3"
```

Comment:
WARNING: [RtlHardcoded] Consider adding `android:layout_marginEnd="@dimen/dp_3"` to better support right-to-left layouts
 Using `Gravity#LEFT` and `Gravity#RIGHT` can lead to problems when a layout is rendered in locales where text flows from right to left. Use `Gravity#START` and `Gravity#END` instead. Similarly, in XML `gravity` and `layout_gravity` attributes, use `start` rather than `left`.

For XML attributes such as paddingLeft and `layout_marginLeft`, use `paddingStart` and `layout_marginStart`. *NOTE*: If your `minSdkVersion` is less than 17, you should add *both* the older left/right attributes *as well as* the new start/right attributes. On older platforms, where RTL is not supported and the start/right attributes are unknown and therefore ignored, you need the older left/right attributes. There is a separate lint check which catches that type of error.

(Note: For `Gravity#LEFT` and `Gravity#START`, you can use these constants even when targeting older platforms, because the `start` bitmask is a superset of the `left` bitmask. Therefore, you can use `gravity="start"` rather than `gravity="left|start"`.)